service_mesh/service_mesh_user_guide/ossm-observability.adoc

```diff
@@ -0,0 +1,27 @@
+[id="ossm-observability"]
+= Data visualization and observability 
+include::modules/ossm-document-attributes.adoc[]
+:context: observability
+
+toc::[]
+
+You can view your application's topology, health and metrics with using the Kiali operator you installed as part of your {ProductName} installation. 
```

Comment:
with using?   Also, you're not using the Operator here, you're using the Kiali application and console _**installed**_ by the operator.